recipe/meta.yaml

```diff
@@ -11,21 +11,20 @@ source:
 
 build:
     number: 0
-    features:
-        - vc9  # [win and py27]
-        - vc10  # [win and py34]
-        - vc14  # [win and py>=35]
-    skip: true  # [win and py>35]
+    run_exports:
+        - {{ pin_subpackage('zlib', max_pin='x.x.x') }}
 
 requirements:
     build:
-        - python  # [win]
-        - cmake  # [win]
-        - msinttypes  # [win and py<35]
+        - cmake       # [win]
+        - msinttypes  # [win and vc<14]
+        - {{ compiler('c') }}  # [win]
```

Comment:
@msarahan, no issues with using the `compiler` functions in windows now right?

Reply:
no issue, but the packages themselves are not true compilers - they are only activation scripts.  Same idea, and same benefits as far as run_exports and all that.  Just different implementation behind the scenes.